api/models.py

```diff
@@ -136,6 +161,7 @@ class PopularHistoryInfo(models.Model):
 
     img_url = models.URLField(max_length=2000, default='')
     description = models.TextField(default='')
+    page = models.ForeignKey(Page, null=True, on_delete=models.SET_NULL)
```

Comment:
nevermind...github collapses the part that shows that this is part of PopularHistoryInfo...